translations.php

```diff
@@ -40,7 +40,7 @@ class BfxTranslations
       'toys' => 'Giocattoli e prodotti per bambini',
       'other' => 'Altro',
       'accepts' => 'Accetta',
-      'accepted_tokens' => 'Token accettati'
+      'accepted_payment_methods' => 'Accepted Payment Methods'
```

Comment:
I've just used google translate
```suggestion
      'accepted_payment_methods' => 'Metodi di pagamento accettati'
```